modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -35,41 +35,109 @@ metadata:
 ----
 <1> If a label has been added it appears under `labels`.
 
-.. If a label is not present, add a key/value pair:
+.. If the label is not present, add a key/value pair:
 +
 ----
 $ oc label machineconfigpool worker custom-kubelet=small-pods
 ----
 
 .Procedure
 
-. Create a Custom Resource (CR) for your configuration change.
+. Configure container garbage collection:
+
+.. Create a Custom Resource (CR) for your configuration change.
 +
-.Sample configuration for a *garbage* CR
+.Sample configuration for a container garbage collection CR:
 [source,yaml]
 ----
 apiVersion: machineconfiguration.openshift.io/v1
 kind: KubeletConfig
 metadata:
-  name: set-garbage-collection <1>
+  name: gc-container <1>
 spec:
-  machineConfigPoolSelector:
+  machineConfigSelector:
     matchLabels:
       custom-kubelet: small-pods <2>
-  kubeletConfig:
-    ImageMinimumGCAge: 0 <3>
-    ImageGCHighThresholdPercent: 85 <4>
-    ImageGCLowThresholdPercent: 80 <5>
-----
-<1> Assign a name to CR.
-<2> Specify the label to apply the configuration change.
-<3> Specify the minimum age for an unused image before it is garbage collected. A value of `0` means no limit.
-<4> Specify the percent of disk usage after which image garbage collection is always run.
-<5> Specify the percent of disk usage before which image garbage collection is never run.
-
-. Create the garbage collection object:
+  kubeletArguments:
```

Comment:
we don't have kubeletArguments anymore in 4.x

Reply:
good catch.  @mburke5678 just remove the `kubeletArguments:` line and we need to reformat the eviction stuff below as well.  replace `eviction-soft` with `EvictionSoft` and `eviction-soft-grace-period` with `EvictionSoftGracePeriod`.

Reply:
@rphillips @sjenning  Is this right?
```
spec:
  machineConfigSelector:
    matchLabels:
      custom-kubelet: small-pods
    EvictionSoft:
    - memory.available<100Mi
    - nodefs.available<10%
    - nodefs.inodesFree<5%
    - imagefs.available<15%
    - imagefs.inodesFree<10%
   EvictionSoftGracePeriod:
    - memory.available=1m30s
    - nodefs.available=1m30s
    - nodefs.inodesFree=1m30s
    - imagefs.available=1m30s
    - imagefs.inodesFree=1m30s
```